docs/.vitepress/theme/style.css

```
@@ -0,0 +1,148 @@
+/* Impeccable CSS baseline for HeliosLab docs theme */
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+html {
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  text-rendering: optimizeLegibility;
```

Stylelint (17.11.1)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)